msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/PrivateMessageManagerImpl.java

```diff
@@ -1186,7 +1186,8 @@ public void sendPrivateMessage(PrivateMessage message, Map<User, Boolean> recipi
 
     message.setRecipients(recipientList);
 
-    savePrivateMessage(message, false);
+    Message savedMessage = savePrivateMessage(message, false);
+    message.setId(savedMessage.getId());
```

Comment:
Is it cleaner or more correct to just begin using savedMessage below this line? Instead of calling message.setId.

Reply:
I saw they are different objects, Message vs PrivateMessage

Reply:
Ahhh, ok